e/etcd/etcd_ubi8.4.sh

```diff
@@ -0,0 +1,63 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : github.com/etcd-io/etcd and go.etcd.io/etcd
+# Version       : v3.5.1
+# Source repo   : https://github.com/etcd-io/etcd
+# Tested on     : RHEL 8.4
```

Comment:
If we have tested this in a UBI container, let's mention the distro as UBI 8.4. If it has been tested on RHEL as well, let's specific both the distros with versions as a comma separated list.

The script name should be "etcd_ubi_8.4.sh" or "etcd_rhel_8.4.sh" accordingly

Reply:
updated distro version to ubi only.